File: graph01/bank_end/ruoyi-system/src/main/java/com/ruoyi/system/domain/Customer.java
package com.ruoyi.system.domain;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import lombok.Builder;
import java.time.LocalDateTime;

/**
 * Customer实体类，对应customer表
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Customer {

    private Integer id;

    private String name;

    private String description;

    private Boolean status;

    private String imageUrl;

    private String createdAt;

    private String updatedAt;
}

File: graph01/graph_fontend/vo/Customer.js
/**
 * Customer VO - 用户实体
 */
export class Customer {
  constructor(data) {
    this.id = data.id || null;
    this.name = data.name || '';
    this.description = data.description || '';
    this.status = data.status || false;
    this.imageUrl = data.imageUrl || '';
    this.createdAt = data.createdAt || '';
    this.updatedAt = data.updatedAt || '';
  }
}
